src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -37,14 +37,16 @@ export default function ColorPickerControl({
     return (
         <div className="bp-ColorPickerControl">
             <button
-                className="bp-ColorPickerControl-button"
+                className={classNames('bp-ColorPickerControl-button', { 'bp-is-toggled': isColorPickerToggled })}
```

Comment:
Should this also be renamed to `bp-ColorPickerControl-toggle`? Same for the swatch?

Reply:
Yes, will rename them